package.json

```diff
@@ -34,6 +34,8 @@
     "prismarine-entity": "0.1.0",
     "prismarine-world": "0.3.3",
     "prismarine-world-sync": "0.1.0",
+    "prismarine-item": "git://github.com/PrismarineJS/prismarine-item.git",
+    "prismarine-windows": "git://github.com/PrismarineJS/prismarine-windows.git",
```

Comment:
The reason I put the repository is because they weren't published when I wrote this.